src/components/AppContent.js

```diff
@@ -1,13 +1,98 @@
 import '../css/App.css';
+import React, { useState } from 'react';
 import KevContainer from './KevContainer.js';
 import QuoteContainer from './QuoteContainer.js';
+import StartButton from './StartButton.js';
+import ItsKevButton from './ItsKevButton.js';
+import NotKevButton from './NotKevButton.js';
+import Points from './Points.js';
+import ResetButton from './ResetButton.js';
+import newQuote from '../utils/quoteGenerator';
+
+const placeholderQuote = {author: "Placeholder", quotation: "Hit the button to start."};
+const endQuote = {author: "Placeholder", quotation: "That's all folks!"};
+let usedQuotes = [];
 
 const AppContent = () => {
+  const [quote, setQuote] = useState(placeholderQuote);
+  const [status, setStatus] = useState("start");
+  const [points, setPoints] = useState(0);
+
+  if (status === "reset") {
+    setQuote(placeholderQuote);
+    setPoints(0);
+    setStatus("open");
+  }
+
+  if (status === "end") {
+    document.getElementById("its-kev-button").setAttribute("style", "display: none;");
+    document.getElementById("not-kev-button").setAttribute("style", "display: none;");
+  }
+
+  const handleStartClick = (event) => {
+    if (usedQuotes.length === 13) {
+      setQuote(endQuote); 
+      setStatus("end");
+      } else {
+          let newQuoteObject;
+          do {
+            newQuoteObject = newQuote();
+          } while (usedQuotes.includes(newQuoteObject.id));
+          usedQuotes.push(newQuoteObject.id);
+          console.log(usedQuotes);
+          setQuote(newQuoteObject);
+          setStatus("open");
+        }
+  }
+
+  const handleResetClick = () => {
+    if (status === "end") {
+      document.getElementById("its-kev-button").setAttribute("style", "display: inital;");
+      document.getElementById("not-kev-button").setAttribute("style", "display: initial;");
+    }
+    setStatus("reset");
+    usedQuotes = [];
+  }
+
+  const handleKevClick = () =>  { 
+    if (quote.author === "Kevin McCloud") { 
```

Comment:
On your point of "how to tell which button clicked it" you could pass a parameter into this function, and then do a check to see if the button that was clicked was the correct answer? So in the component: 
`<Button onClick={() => handleClick("kev")}>`
which would look like:
```
const handleClick = (buttonPressed) => {
 ... logic for checking
}
```

What do you think?

Reply:
Ahh yep!! I've done that in ac9aa1b just now. Thanks!